src/domain/safe/entities/multisig-transaction.entity.ts

```diff
@@ -0,0 +1,42 @@
+export type Call = 0;
+export type Delegate = 1;
+export type Operation = Call | Delegate;
+
+export interface Confirmation {
+  owner: string;
+  submissionDate: string; // TODO check Date type validation
+  transactionHash?: string;
+  signatureType: string;
+  signature?: string;
+}
+
+export interface MultisigTransaction {
+  safe: string;
+  to: string;
+  value?: string;
+  data?: string;
+  dataDecoded?: any; // TODO will be added under https://github.com/5afe/safe-client-gateway-nest/pull/132
```

Comment:
Nitpick: this doesn't matter since it will be replaced soon as the TODO says, but it seems generally preferable to use `unknown` instead of `any`. 
Links:
https://type-level-typescript.com/02-types-are-just-data#what-about-any
https://type-level-typescript.com/02-types-are-just-data#unknown--the-top-of-the-subtyping-hierarchy

Reply:
Yeah I agree 👍 Changed here [53608a8](https://github.com/5afe/safe-client-gateway-nest/pull/135/commits/53608a8f8d67361a4d5e0fa678bbdfc0fd2be217)

Reply:
Had to revert due to typescript incompatibilities [5c29d59](https://github.com/5afe/safe-client-gateway-nest/pull/135/commits/5c29d599eb2548f29a96266b92b3babff84bf97f)